xss_scanner/requirements.txt

```diff
@@ -0,0 +1,2 @@
+splitner
```

Comment:
use requirements with versions - this way if a breaking change happens on the library side, we wouldn't silently stop working

add this directory to Dependabot